lib/router/package.json

```diff
@@ -37,7 +37,8 @@
     "*.d.ts"
   ],
   "scripts": {
-    "prepare": "ts-node ../../scripts/prebundle.ts"
+    "prepare": "../../node_modules/.bin/ts-node ../../scripts/prepare-prebundle.ts",
+    "prepublish": "../../node_modules/.bin/ts-node ../../scripts/prepare-prebundle.ts --optimized"
```

Comment:
this script is invoked (automatically by npm itself) before publish, generating a production version of the `dist` dir

Reply:
why was ts-node removed and `../../node_modules` was used instead?

Reply:
less dependencies in package.json
fewer places where dis-organisation (many different versions) could come up.

Reply:
🤔  shouldn't the package just be hoisted in the monorepo? In that case the reference should be the same, right?

Reply:
> 🤔 shouldn't the package just be hoisted in the monorepo? In that case the reference should be the same, right?
yes, and yes.

But it will still be in package.json. Thus in yarn.lock
And if we update in 1 place, but not the other, that's pain.

I'm not sure why you think having it as a devDependency is better?

Reply:
perhaps this is an argument for @ethriel3695's `storybook-scripts` idea